dbaas-cli/cmd/mongo/create.go

```diff
@@ -71,12 +79,19 @@ var createCmd = &cobra.Command{
 				//log.Error("check db: ", err)
 				continue
 			}
-			if cluster.Status == "ready" {
+			switch cluster.Status {
+			case "ready":
 				dotPrinter.Stop("done")
 				log.Println("Database started successfully, connection details are below:")
 				cluster.Message = strings.Replace(cluster.Message, "PASSWORD", cluster.Pass, 1)
-				log.Println(cluster)
+				log.WithField("database", cluster).Info("information")
 				return
+			case "initializing":
+				if noWait {
+					cluster.Message = strings.Replace(cluster.Message, "PASSWORD", cluster.Pass, 1)
+					log.WithField("database", cluster).Info("information")
```

Comment:
if the output format is JSON, should be the cluster printed in as JSON?

Reply:
yes